src/main/kotlin/no/skatteetaten/aurora/boober/service/OpenShiftCommandService.kt

```diff
@@ -184,6 +184,7 @@ class OpenShiftCommandService(
 
         val labelSelectors = listOf("app=$name", "booberDeployId", "booberDeployId!=$deployId")
         return apiResources
+            .filter { kind -> !kind.toLowerCase().equals("auroracname") || openShiftClient.k8sVersionOfAtLeast("1.16") }
```

Comment:
Vil ikke dette (`openShiftClient.k8sVersionOfAtLeast("1.16")`) føre til at ingenting blir slettet på OCP3?

Reply:
Nei, skulle ikke tro det. Dersom kind _ikke_ er lik "auroracname" går den videre.  Dersom den er lik, går den videre dersom du er på OCP4. Det er en || og ikke en && i uttrykket.

Reply:
Bare dårlig formulert fra min side. Er det slik at det er mulig å opprette AuroraCname på OCP3, men det skal ikke være mulig å slette den derfra?

Reply:
Nei, det er ikke mulig å opprette auroracname på OCP3, fordi denne entiteten ikke finnes. K8s fikk ikke støtte for CRDer før versjon 1.16, og på OCP3 kjører vi 1.11.